# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../vice/compile-fix.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- vice-2.1.orig/src/arch/unix/x11/xaw/x11video.c
+++ vice-2.1/src/arch/unix/x11/xaw/x11video.c
@@ -59,6 +59,8 @@
 #include <X11/Xutil.h>
 #include <X11/Intrinsic.h>
 #include <X11/cursorfont.h>
+#include <X11/Xmd.h>
+#include <X11/extensions/shmproto.h>
 
 #include <stdio.h>
 #include <stdlib.h>

